frontend/cypress/integration/admin/admin-smoke-tests.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  aliasMutation,
+  aliasQuery,
+  setUpCommonGraphqlIntercepts,
+} from "../../support/graphql-test-utils";
+import { getInputByLabel } from "../../support/helpers";
+
+describe("Admin Smoke Tests", () => {
+  const loginViaUI = (role) => {
+    cy.fixture("users.json").then((users) => {
+      const user = users[role];
+      cy.get("input[name=username]").type(user.email);
+    });
+    cy.findByText("Sign-in").click();
+  };
```

Comment:
Unless I'm misunderstanding how "smoke test" might be different, I'd propose we just remove this and use `cy.login('admin')` below

The idea with E2E tests is usually that the UI is only tested once (in this case, in `auth.test.js`), and then after that things are done with more efficient commands (that ideally use `cy.request` directly) and are no longer exercising the UI (since that's already covered).